client/src/components/sources/sources.js

```diff
@@ -96,16 +96,20 @@ class Sources extends React.Component {
 
   showAddSourceWizard() {
     Store.dispatch({
-      type: sourcesTypes.EDIT_SOURCE_SHOW
+      type: sourcesTypes.CREATE_SOURCE_SHOW
+    });
+  }
+
+  editSource(item) {
+    Store.dispatch({
+      type: sourcesTypes.EDIT_SOURCE_SHOW,
+      source: item
     });
   }
 
   importSources() {
     Store.dispatch({
-      type: toastNotificationTypes.TOAST_ADD,
-      alertType: 'error',
-      header: 'NYI',
-      message: 'Importing sources is not yet implemented'
+      type: sourcesTypes.IMPORT_SOURCE_SHOW
```

Comment:
I believe this feature was removed. It's no longer in the mocks for the sources empty state. Maybe only part of the wizard now?

Reply:
And now the import option is now longer there